src/engine/loaders/obj_loader.cpp

```diff
@@ -71,31 +71,140 @@ void ObjLoader::load(MeshFrame *o_result, char *t_filename, float t_scale, u8 t_
             }
             else if (strcmp(lineHeader, "f") == 0)
             {
-                int matches = fscanf(file, "%d/%d/%d %d/%d/%d %d/%d/%d\n",
-                                     &vertexIndex[0], &coordIndex[0], &normalIndex[0],
-                                     &vertexIndex[1], &coordIndex[1], &normalIndex[1],
-                                     &vertexIndex[2], &coordIndex[2], &normalIndex[2]);
-                if (matches != 9)
-                    PRINT_ERR(".obj can't be read by this simple parser. Try exporting with other options");
+
+                int* x;
+                fpos_t start;
+                fgetpos(file,&start);
+                        int matches = fscanf(file, "%d/%d/%d %d/%d/%d %d/%d/%d\n",x,x,x,x,x,x,x,x,x);
+                fsetpos(file,&start);
```

Comment:
I see that you probably don't using autoformatter. 
Please use it to prevent formatting IDE battles.

Reply:
Done.